modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/resources/com/liferay/dynamic/data/mapping/dependencies/upgrade.v5_3_2/ddm-template-browser-sniffer-content.ftl

```diff
@@ -0,0 +1,24 @@
+<#--
+Web content templates are used to lay out the fields defined in a web
+content structure.
+Please use the left panel to quickly add commonly used variables.
+Autocomplete is also available and can be invoked by typing "${".
+-->
+
+<#assign normalizer = serviceLocator.findService("com.liferay.portal.kernel.util.FriendlyURLNormalizerUtil")/>
+
+<#assign browserSnifferCaseOne = serviceLocator.findService("com.liferay.portal.kernel.servlet.BrowserSnifferUtil") />
+
+<#assign browserSnifferCaseTwo = serviceLocator.findService("com.liferay.portal.kernel.servlet.BrowserSnifferUtil") >
+
+<#assign
+browserSnifferCaseThree = serviceLocator.findService("com.liferay.portal.kernel.servlet.BrowserSnifferUtil")
+variableA = "variableACaseThree"
```

Comment:
@alberttgomes, please see https://github.com/natocesarrego/liferay-portal/pull/1305/files/c0325e1966ed461be6195bab1d5c884ea9b6abd0..0e863106b19902d167c5cd90ed38dbc5e84a5503#diff-e576ea23720c5da974f82e74e8f84b54c481140088fe6070911dd54e98f8cdb7 and https://github.com/natocesarrego/liferay-portal/pull/1305/files/c0325e1966ed461be6195bab1d5c884ea9b6abd0..0e863106b19902d167c5cd90ed38dbc5e84a5503#diff-e576ea23720c5da974f82e74e8f84b54c481140088fe6070911dd54e98f8cdb7